docs/README.md

```diff
@@ -1,3 +1,83 @@
 # K8ssandra.io - Website & Documentation Repository
 
-See the common docs README for more information [here](https://github.com/k8ssandra/k8ssandra/tree/docs#readme).
+This project is built with the [Hugo](https://gohugo.io/) static site generator and the [Docsy](https://github.com/google/docsy) theme.
+
+The documentation produced from this repository can be viewed at:
+
+Development Version -- https://docs.k8ssandra-dev.io
```

Comment:
I'm not sure how we can access this 🤔 I'm getting an "Access Denied" error message.
Maybe we should just mention https://docs-staging.k8ssandra.io` instead?

I mean, our workflow will really be to start hugo locally for dev purposes, and then once a PR is merged it's the staging website which will be updated.

I'm not sure how that `k8ssandra-dev.io` website fits into that workflow.

Reply:
I will update this.

Reply:
done